tables/isolates.csv

```diff
@@ -1302,3 +1302,10 @@ B.1.640.2 Spike,B.1.640.2,FALSE,7314471,NULL,NULL,TRUE
 hCoV-19/South Africa/CERI-KRISP-K040015/2022,Omicron/BA.4,FALSE,12268495,NULL,NULL,TRUE
 hCoV-19/South Africa/CERI-KRISP-K040013/2022,Omicron/BA.5,FALSE,12268493,NULL,NULL,TRUE
 hCoV-19/South Africa/CERI-KRISP-K032888/2021,Omicron/BA.1,FALSE,7886688,NULL,NULL,TRUE
+BA.1 Spike:+376A+405N+408S+452R-446S+486V-493R-496S-547K-856K-981F,Omicron/BA.1,FALSE,NULL,NULL,NULL,TRUE

```

Comment:
This is a custom pseudovirus that combines BA.1 and BA.4/5 mutations, I wasn't sure how to properly record this

Reply:
The name is ok, just list all the mutations you have, and please use NULL for the `variant name`.